web-gateway/app/views/item.scala.html

```diff
@@ -93,11 +93,21 @@
         }
     }
     </dl>
-
-    @if(currentUser == item.creator && item.status == ItemStatus.Created) {
-        <form action="@routes.ItemController.startAuction(item.safeId)" method="POST">
-            <input class="button success" type="submit" value="@Messages("startAuction")"/>
-        </form>
+    @if(currentUser == item.creator) {
+        <div class="row">
+            @if(item.status == ItemStatus.Created) {
+            <div class="columns medium-6" >
+                <form action="@routes.ItemController.startAuction(item.safeId)" method="POST">
+                    <input class="button success" type="submit" value="@Messages("startAuction")"/>
+                </form>
+            </div>
+            }
+            @if(! Seq(ItemStatus.Completed, ItemStatus.Cancelled).contains(item.status)) {
+            <div class="columns medium-6" >
+                <a href="@routes.ItemController.updateItemForm(item.safeId)" class="button primary">@Messages("updateItem")</a>
+            </div>
+            }
+        </div>
     }
```

Comment:
the edit page is a bit more tricky. I understand that it's WIP as is is now

Remember that when the auction already started only the description may be edited.